components/script/build.rs

```diff
@@ -47,7 +47,7 @@ fn main() {
     let mut phf = File::create(&phf).unwrap();
     write!(
         &mut phf,
-        "pub static MAP: phf::Map<&'static [u8], unsafe fn(*mut JSContext, HandleObject)> = "
+        "pub static MAP: phf::Map<&'static [u8], /* unsafe */ fn(/* *mut */ JSContext, HandleObject)> = "
```

Comment:
@jdm same here. I commented `unsafe` and `*mut` for now since we accept safe wrapped JSContext now.

(sorry for the mess, I should've removed those)